pyk/src/pyk/rpc/rpc.py

```diff
@@ -96,7 +98,7 @@ class JsonRpcError(JsonRpcResult):
     message: str
     id: str | int | None
 
-    def to_json(self) -> dict[str, Any]:
+    def wrap_response(self) -> dict[str, Any]:
```

Comment:
The PR description states

> This PR extends the JsonRpcServer with support for streaming.

but it introduces what look like breaking changes. Are there other clients to this library other than `kontrol-node`?